type.go

```diff
@@ -399,6 +402,48 @@ func (t int32Type) AssignValue(dst reflect.Value, src Value) error {
 		dst.SetInt(int64(v))
 	case reflect.Uint8, reflect.Uint16, reflect.Uint32:
 		dst.SetUint(uint64(v))
+	case reflect.Pointer:
+		underlying := reflect.TypeOf(dst).Elem().Kind()
+		switch underlying {
+		case reflect.Float32:
+			val := float32(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Float64:
+			val := float64(v)
+			dst.Set(reflect.ValueOf(&val))
```

Comment:
We're introducing new conversions from float to int that did not exist before; I would prefer if we removed these so there are no implicit conversions between values of fundamentally different types.

Reply:
I agree, I removed the conversions.